src/pages/register-a-beacon/about-the-vessel.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  Form,
+  FormFieldset,
+  Input,
+  FormLegendPageHeading,
+  FormLabel,
+  FormGroup,
+  FormHint,
+} from "../../components/Form";
+import { IfYouNeedHelp } from "../../components/Mca";
+import { GetServerSideProps, GetServerSidePropsContext } from "next";
+import { withCookieRedirect } from "../../lib/middleware";
+import { TextAreaCharacterCount } from "../../components/TextArea";
+
+const CheckBeaconDetails: FunctionComponent = (): JSX.Element => {
+  return (
+    <>
+      <Layout
+        navigation={<BackButton href="/register-a-beacon/primary-beacon-use" />}
+      >
+        <Grid
+          mainContent={
+            <>
+              <Form action="/register-a-beacon/about-the-vessel">
+                <FormFieldset>
+                  <FormLegendPageHeading>
+                    About the pleasure vessel
+                  </FormLegendPageHeading>
+
+                  <MaxCapacityInput />
+
+                  <VesselNameInput />
+
+                  <HomeportInput />
+
+                  <AreaOfOperationTextArea />
+
+                  <BeaconLocationInput />
+                </FormFieldset>
+                <Button buttonText="Submit" />
+              </Form>
+              <IfYouNeedHelp />
+            </>
+          }
+        />
+      </Layout>
+    </>
+  );
+};
+
+const MaxCapacityInput: FunctionComponent = (): JSX.Element => (
+  <FormGroup>
+    <FormLabel htmlFor="maxCapacity">
+      Enter the maximum number of persons aboard
+    </FormLabel>
+    <FormHint forId="maxCapacity">
+      Knowing the maximum number of persons likely to be onboard the vessel
+      helps Search & Rescue know how many people to look for and what resources
+      to send
+    </FormHint>
+    <Input
+      name="maxCapacity"
+      id="maxCapacity"
+      numOfChars={5}
+      htmlAttributes={{
+        spellCheck: "false",
+        "aria-describedby": "maxCapacity-hint",
+        pattern: "[0-9]*",
+        inputMode: "numeric",
+      }}
+    />
+  </FormGroup>
+);
+
+const VesselNameInput: FunctionComponent = (): JSX.Element => (
+  <FormGroup>
+    <FormLabel htmlFor="vesselName">
+      Enter your vessel name (optional)
+    </FormLabel>
+    <Input
+      name="vesselName"
+      id="vesselName"
+      htmlAttributes={{ spellCheck: false }}
+    />
+  </FormGroup>
+);
+
+const HomeportInput: FunctionComponent = (): JSX.Element => (
+  <FormGroup>
+    <FormLabel htmlFor="homeport">
+      Enter the Homeport for the vessel (optional)
+    </FormLabel>
+    <FormHint forId="homeport">
+      This is the name of the port where your vessel is registered and primarily
+      operates from
+    </FormHint>
+    <Input
+      name="homeport"
+      id="homeport"
+      htmlAttributes={{ spellCheck: false }}
+    />
+  </FormGroup>
+);
+
+const AreaOfOperationTextArea: FunctionComponent = (): JSX.Element => (
+  <TextAreaCharacterCount
+    name="areaOfOperation"
+    id={"areaOfOperation"}
```

Comment:
You shouldn't need to use the curly braces here, can just be: `id="areaOfOperation"`

Reply:
🙈 @stuart-madetech had to say the same many times when we were pairing, I just love those brace 😂